src/utils.rs

```diff
@@ -427,7 +430,7 @@ pub(crate) async fn launch(
         if wrapper_vec.len() > 0 {
             wrapper_vec[0].to_owned()
         } else {
-            if should_use_wine {
+            if !no_wine {
```

Comment:
This is wrong for macOS. The previous implementation with `should_use_wine` is the correct one. We don't want to use Wine when running a native macOS game.